gradle.properties

```diff
@@ -2,14 +2,12 @@
 org.gradle.jvmargs=-Xmx1G
 
 minecraft_version=1.14.4
-yarn_mappings=1.14.4+build.15
-loader_version=0.7.0+build.171
+yarn_mappings=1.14.4+build.18
+loader_version=0.14.8
 
-fabric_version=0.4.2+build.246-1.14
-fabric_asm_version=b97939c03a
-zt_zip_version=1.13
-slf4j_version=1.7.29
+fabric_version=0.28.5+1.14
+fabric_asm_version=v2.3
 
-mod_version = 0.7.1
+mod_version = 0.7.5
```

Comment:
Think it might be worth bumping this to 0.8.0

Reply:
Sure, sounds good, I've bumped the version